docs/elements.md

```diff
@@ -608,6 +610,61 @@ Attribute | Type | Default | Description
 
 -----
 
+### `pl-matching` element
+
+Given a list of questions, select a matching option for each entry from a drop-down list.
+
+#### Sample element
+
+![](elements/pl-matching.png)
+
+**question.html**
+```html
+<pl-matching answers-name="string_value">
+  <pl-answer match="d.c.">United States</pl-answer>
+  <pl-answer match="Mexico City">Mexico</pl-answer>
+  <pl-answer match="Paris">France</pl-answer>
+
+  <pl-option name="d.c.">Washington, D.C.</pl-option>
+  <pl-option>Mexico City</pl-option>
+  <pl-option>Paris</pl-option>
+</pl-matching>
+```
+
+#### Customizations
+
+Attribute | Type | Default | Description
+--- | --- | --- | ---
+`answers-name` | string | — | Variable name to store data in.
+`weight` | integer | 1 | Weight to use when computing a weighted average score over elements.
+`fixed-order` | boolean | False | Whether or not to display the answers in a fixed order; otherwise they are shuffled. Options are always shuffled.
+`number-answers` | integer | special | The number of answers to display. Defaults to all answers.
+`number-options` | integer | special | The number of options to display. Defaults to all options.
+`none-of-the-above` | boolean  | false | Whether or not to add a "None of the above" to the end of the options. Automatically set to true if number-options is less than the total number of options.
+`blank` | boolean | False | Option to add blank dropdown entry as the default selection in each drop-down list..
+`counter-type` | "decimal" or "lower-alpha" or "upper-alpha" | "decimal" | The type of counter to use when enumerating the options.
```

Comment:
I'd love to see an option where what's shown is the option text itself (either the name or the text). This can be useful when there is a limited number of options. For example, I have a question where students match to "static" or "dynamic", and listing these instead of 1/2 or a/b is probably less error-prone to students. I don't know if this would be a separate option (e.g., `include-option-text` or something like that) or replace the counter in that case, let me know what you think.